### PR TITLE
[Manager] Fix connection to second local client instance

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

**Comment by RichardHaselgrove** [comment]
I've tested that the presenting problem is fixed. I need to look back into why we made the previous change, and check that solution is still valid - then I'll post a formal review.

**Comment by RichardHaselgrove** [comment]
Actually, there is one side-effect. I also use (under Windows) the Manager file menu function 'New BOINC Manager window...' (which is an alternative route into using multiple managers). In this use case, the intention is to use the new (additional) window to monitor a different, remote, computer.

Previously the new window opened blank, with the 'Select Computer' dialog showing. Now, it opens with the local client displayed, and I have to make a second mouse movement to 'Select Computer'. Ooh, the hardship! Better that both use-cases still work, with a little extra effort, than that one can't be used. But if....

**Comment by AenBleidd** [comment]
Thank you. I'll fix this too

**Comment by AenBleidd** [comment]
I made a 'blind' fix but didn't verify it yet. Plan to verify it on Monday.

**Comment by AenBleidd** [comment]
@RichardHaselgrove, should be fixed now. Please, verify it again.

**Comment by Germano0** [comment]
Hi Vitalii, I think it would be great to have a full description of the problem that this patch is fixing, so that the testing process can be more accurate.
Thank you

**Comment by RichardHaselgrove** [comment]
@Germano0: It arises from the email I sent to the boinc_alpha email list at 21 Oct at 20:12 (UTC+1). Details are there.

File: clientgui/MainDocument.cpp [hunk 1]
```diff
@@ -583,6 +583,7 @@ int CMainDocument::OnPoll() {
     int iRetVal = 0;
     wxString hostName = wxGetApp().GetClientHostNameArg();
     wxString password = wxGetApp().GetClientPasswordArg();
+	bool isHostnamePasswordSet = wxGetApp().IsHostnamePasswordSet();
```
Comment (TheAspens) [inline clientgui/MainDocument.cpp]: @AenBleidd - can you fix the indentation here?

**Comment by AenBleidd** [comment]
Damn, I made a force-push of incorrect commit. Will fix it later with the indentation fix requested above

**Comment by AenBleidd** [comment]
Damn, I made a force-push of incorrect commit. Will fix it later with the indentation fix requested above

**Comment by AenBleidd** [comment]
Ok, now it's fine.
FYI: https://objectpartners.com/2014/02/11/recovering-a-commit-from-githubs-reflog/

**Comment by AenBleidd** [comment]
No, in [this](https://groups.google.com/a/ssl.berkeley.edu/forum/#!topic/boinc_alpha/aH_izVE7kRg)

**Comment by RichardHaselgrove** [comment]
> Is it this discussion?

@AenBleidd has the correct thread.

**Comment by TheAspens** [comment]
Based on the discussion on the contributor call this morning and the code and testing it looks good.  @AenBleidd thanks for the fix!

**Pull request merged**